app/src/main/res/values/strings.xml

```diff
@@ -113,7 +113,7 @@
     <string name="desc_coingecko">Ergo price powered by CoinGecko API, https://www.coingecko.com</string>
     <string name="label_cg_conn_error">Error connecting to CoinGecko, please try again later.</string>
     <string name="desc_dark_mode">Dark or light mode?</string>
-    <string name="button_dark_mode_system">Follow system</string>
+    <string name="button_dark_mode_system">Respect system</string>
     <string name="button_dark_mode_day">Light</string>
     <string name="button_dark_mode_night">Dark</string>
```

Comment:
Thank you for your improvements! All look very good to me and I will merge when I checked if everything fits when shown in the app.

I have just a question regarding the change "Follow system" to "Respect system": The option that is set is actually named that way in internal Android documentations (not shown to end users).
When I now checked how the option is named in other apps, I found "System default" to be the best for end users to understand what the option actually does - do you agree?

Reply:
Ah, I agree! System default is indeed better.